### PR TITLE
Save the children !

### PR DESCRIPTION
**Pull request by fyusuf-a**
A back seat is interesting for underage passengers. I have tried in this commit :
* a back seat on the car
* still a front seat

### CODE REVIEW

File: my_car.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 - front seat
+- back seat
+- 4 wheels
```
Comment (Mel-louie) [inline my_car.txt:3]:
```suggestion
- 4 wheels and a spare
```

Comment (fyusuf-a) [reply my_car.txt:3]: But a spare is expansive, isn't it?

File: my_car.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 - front seat
+- back seat
```
Comment (Mel-louie) [inline my_car.txt:2]: good idea

File: my_car.txt (hunk 1 above)
Comment (fyusuf-a) [reply my_car.txt:3]: I really want to save the children so nevermind.

**Review by Mel-louie — APPROVED** [review]

**Pull request merged**